packages/beacon-node/src/chain/validation/blobSidecar.ts

```diff
@@ -207,3 +217,17 @@ function validateBlobsAndProofs(
     throw Error("Invalid verifyBlobKzgProofBatch");
   }
 }
+
+function validateInclusionProof(config: ChainForkConfig, blobSidecar: deneb.BlobSidecar): void {
+  if (
+    !verifyMerkleBranch(
+      ssz.deneb.KZGCommitment.hashTreeRoot(blobSidecar.kzgCommitment),
+      blobSidecar.kzgCommitmentInclusionProof,
+      KZG_COMMITMENT_INCLUSION_PROOF_DEPTH,
+      KZG_COMMITMENT_SUBTREE_INDEX0 + blobSidecar.index,
+      blobSidecar.signedBlockHeader.message.bodyRoot
+    )
+  ) {
+    throw Error("Invalid next sync committee merkle branch");
```

Comment:
This error message is not correct. Also why do the try catch wrapping here and return a bool?